src/main/java/sirius/search/Query.java

```diff
@@ -1165,6 +1166,8 @@ private QueryBuilder buildQuery() {
         if (Microtiming.isEnabled()) {
             w.submitMicroTiming("ES", "LIST: " + toString(true));
         }
+        indexAccess.addSelectTiming(w.elapsedMillis());
```

Comment:
wouldnt it make more sense to do this in SearchRequestBuilder.execute().actionGet() ?